src/main/java/seedu/sellsavvy/logic/commands/personcommands/AddPersonCommand.java

```diff
@@ -40,6 +40,12 @@ public class AddPersonCommand extends Command {
 
     public static final String MESSAGE_SUCCESS = "New person added: %1$s";
     public static final String MESSAGE_DUPLICATE_PERSON = "This person already exists in the address book.";
+    public static final String MESSAGE_SIMILAR_TAGS_WARNING = "Note: "
```

Comment:
Since this is being used at EditPersonCommand, can this be moved to Messages.java instead?

Reply:
similar tags warning is different.